File: ObjcCrashCourse/InMyHoodObjC/InMyHoodObjC/PostCell.m
//
//  PostCell.m
//  InMyHoodObjC
//
//  Created by Allen Czerwinski on 5/2/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

#import "PostCell.h"

@implementation PostCell

- (void)awakeFromNib {
    [super awakeFromNib];
    
    myDataService = [DataService sharedInstance];
    
    int frameSize = _postImg.frame.size.width / 2;
    [[_postImg layer] setCornerRadius:frameSize];
    _postImg.clipsToBounds = YES;
    
}

-(void)configureCell:(Post *)post {
    
    [_titleLabel setText:post.title];
    [_descriptionLabel setText:post.postDesc];
    [_postImg setImage:[myDataService imageForPath:post.imagePath]];
    
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: Swift2iOS9Udemy/My Hood App/My Hood App/PostCell.swift
//
//  PostCell.swift
//  My Hood App
//
//  Created by Allen Czerwinski on 2/17/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

import UIKit

class PostCell: UITableViewCell {
    
    @IBOutlet weak var postImg: UIImageView!
    @IBOutlet weak var titleLbl: UILabel!
    @IBOutlet weak var descLbl: UILabel!

    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }
    
    func configureCell(post: Post) {
        titleLbl.text = post.title
        descLbl.text = post.postDesc
    }
    
    

}
